test/AutoRest.TestServer.Tests/NumberTest.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using AutoRest.TestServer.Tests.Infrastructure;
+using body_number;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class NumberTest : TestServerTestBase
+    {
+        public NumberTest(TestServerVersion version) : base(version, "number") { }
+
+        [Test]
+        public Task putFloatBigScientificNotation() => TestStatus(async (host, pipeline) => await NumberOperations.PutBigFloatAsync(ClientDiagnostics, pipeline, 3.402823e+20f, host));
```

Comment:
Do we care if the test names use camelCase versus PascalCase?

Reply:
Yes, I'll fix